_rules/image-no-text-0va7u6.md

```diff
@@ -36,24 +36,23 @@ This rule applies to any [visible][] [embedded image][].
 
 For the rendered image resource of the [image sources][] of each test target, at least one of the following is true:
```

Comment:
I think we should hoist the visible pixels thing into the setup to avoid repeating that phrase 3 times:

```suggestion
For the [visible pixels][] of the rendered image resource of the [image sources][] of each test target, at least one of the following is true:
```

Reply:
The issue is that there is a 4th condition...